workflows/variantcall.nf

```diff
@@ -41,37 +39,23 @@ workflow VARIANTCALL {
     //     params.chain_checksum
     // )
 
-    // SanityCheck
-    sanityCheck(
+    // Prepare metadata, adapted from SanityCheck
+    PREP_META(
         file(params.experiment_info_tsv),
         params.api_token,
         params.song_url,
         params.clinical_url,
         params.skip_duplicate_check
```

Comment:
Wrong location for it but add default value for `skip_duplicate_check` and edit score version to `5.10.1`